teeworlds7/client.go

```diff
@@ -80,56 +84,66 @@ func (client *Client) SnapFindCharacter(clientId int) (character *object7.Charac
 }
 
 func NewClient() *Client {
-	client := &Client{}
-	client.SnapshotStorage = snapshot7.NewStorage()
-	client.Game.Snap = &GameSnap{}
-	client.Game.Input = &messages7.Input{}
-	client.LocalClientId = -1
-	client.LastSend = time.Now()
-	return client
+	return &Client{
+		SnapshotStorage: snapshot7.NewStorage(),
+		Game: Game{
+			Snap:  &GameSnap{},
+			Input: &messages7.Input{},
+		},
+		LocalClientId: UnknownClientId,
+		LastSend:      time.Now(),
+	}
 }
 
-func (client *Client) sendInputIfNeeded() bool {
-	diff := time.Now().Sub(client.LastSend)
+func (client *Client) sendInputIfNeeded() (sent bool, err error) {
 	send := false
 	// at least every 10hz or on change
-	if diff.Microseconds() > 1000000 {
+	// before, this was set to 1 second
+	if time.Since(client.LastSend) > 100*time.Millisecond {
```

Comment:
here, this value was previously set to 1 second, I changed it inorder to correspond o the comment.

Reply:
The 10hz comment comes from the teeworlds code. The 1 second code was wrong. No need to keep a comment about it thats just confusing.

https://github.com/teeworlds/teeworlds/blob/c56fa9e6a20cfc9d7d16502e18c7d7633acdf492/src/game/client/components/controls.cpp#L183-L185

Reply:
is removed, was there for you.